src/components/Nav/Nav/Nav.scss

```
@@ -0,0 +1,233 @@
+* {
+  box-sizing: border-box;
+}
+header {
+  position: relative;
+  width: 100%;
+  height: 110px;
+  z-index: 50;
+  left: 0;
+  top: 0;
+  // border: 1px solid black;
+  .top_menu {
+    width: 100%;
+    height: 40px;
+    //   border: 1px solid black;
+    background: #e5e5e5;
+    .inner {
+      width: 1200px;
+      margin: 0 auto;
+      .mynb {
+        display: block;
+        float: left;
+        margin-top: 14px;
+      }
+      .made {
+        display: inline-block;
+        margin-top: 12px;
+        margin-left: 20px;
```

- margin 혹은 padding 속성을 한 줄이상 부여하실 땐, 한줄 속성 이용해주세요!
```suggestion
        margin: 12px 0 0 20px;
```